build-test.cmd

```diff
@@ -104,6 +104,13 @@ call %__ProjectDir%\init-tools.cmd
 
 REM =========================================================================================
 REM ===
+REM === Deploy debugger tests
```

Comment:
i believe it is fine for you to move this to runtest, It will keep everything consistent and will be straight forward when we do Cross Build and test, where we build on one machine and copy the output to another to run it [ Think build on x-64 and run on ARM/ARM64].